src/main/kotlin/org/jdc/template/inject/AppModule.kt

```diff
@@ -85,6 +88,12 @@ class AppModule(private val application: Application) {
     fun provideMainDatabase(application: Application): MainDatabase {
         return Room.databaseBuilder(application, MainDatabase::class.java, MainDatabase.DATABASE_NAME)
 //                .addMigrations(object: Migration(1, 2) {
+//                    override fun migrate(database: SupportSQLiteDatabase) {
```

Comment:
With the change of the database to version 2..... this migration should now work (uncomment)